packages/core/src/RenderingEngine/VideoViewport.ts

```diff
@@ -526,6 +582,21 @@ class VideoViewport extends Viewport implements IVideoViewport {
     }
   }
 
+  public getCurrentImageId() {
+    const current = this.imageId.replace(
+      '/frames/1',
+      this.isPlaying
+        ? `/frames/1-${this.numberOfFrames}`
+        : `/frames/${this.getFrameNumber()}`
+    );
+    return current;
+  }
```

Comment:
please add doc on why we do the 1-numberOfFrames, you mentioned it is DICOM way of doing this as well. Function lack documentation